### PR TITLE
Make it possible to regenerate FFMPEG_jll

### PR DESCRIPTION
**Pull request by giordano**

### CODE REVIEW

File: recursively_regenerate_jlls.jl [hunk 1]
```diff
@@ -199,9 +204,9 @@ push!(deps, toplevel_dep_name)
 println("Discovered dependencies: $(collect(deps))")
 
 if yn_prompt(WizardState(), "Open JLL-bumping PRs?", :y) == :y
-	for dep in deps
-		open_jll_bump_pr(dep)
-	end
+    for dep in deps
+	open_jll_bump_pr(dep)
+    end
```
Comment (staticfloat) [inline recursively_regenerate_jlls.jl:209]: tab/space issues?

Comment (giordano) [reply recursively_regenerate_jlls.jl:209]: Yes

**Review by KristofferC — APPROVED** [review]

**Comment by KristofferC** [comment]
I think things still error if e.g. FFMPEG are generated as a part of something else:

```
~/JuliaPkgs/Yggdrasil mg/regenerate-ffmpeg* 10s
❯ julia --project recursively_regenerate_jlls.jl GR
[ Info: Parsing /home/kc/JuliaPkgs/Yggdrasil/G/GR/build_tarballs.jl
...
[ Info: Parsing /home/kc/JuliaPkgs/Yggdrasil/X/Xorg_xtrans/build_tarballs.jl
[ Info: Parsing /home/kc/JuliaPkgs/Yggdrasil/X/Xorg_libXrender/build_tarballs.jl
[ Info: Parsing /home/kc/JuliaPkgs/Yggdrasil/L/LZO/build_tarballs.jl
┌ Warning: Unable to find build_tarballs.jl file for dependency "FFMPEG"
└ @ Main ~/JuliaPkgs/Yggdrasil/recursively_regenerate_jlls.jl:31
[ Info: Parsing /home/kc/JuliaPkgs/Yggdrasil/F/FFMPEG/build_tarballs.jl
ERROR: LoadError: could not open file /home/kc/JuliaPkgs/Yggdrasil/F/FFMPEG/build_tarballs.jl
```

**Comment by giordano** [comment]
Yeah, but I don't know how to make this work reliably :disappointed:

**Comment by KristofferC** [comment]
Why are there subdirectories?

**Comment by giordano** [comment]
We put multiple builders in subdirectories when they share a lot of code, often a somewhat long or complicated build `script`.  [OpenBLAs](https://github.com/JuliaPackaging/Yggdrasil/tree/4f19703ad78e3137a95134c1344beae5d32bfe83/O/OpenBLAS) is a notable case.

**Pull request merged**